ansible/group_vars/renderingservice.yml

```diff
@@ -20,4 +20,4 @@ esrender_exec_method: http
 esrender_archive_retrieve_method: download
 esrender_local_archive_path:
 
-install_splash: false # install splash on renderingservice-hosts - see https://splash.readthedocs.io/en/stable/index.html
+install_splash: false # install splash on renderingservice-hosts - see https://splash.readthedocs.io/en/stable/index.html
```

Comment:
```suggestion
install_splash: false # install splash on renderingservice-hosts - see https://splash.readthedocs.io/en/stable/index.html

```

Reply:
we should not change the format of the file, if there is no functional change at all